src/Plugin/StatisticsPlugin/Dummy.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\sapi\Plugin\StatisticsPlugin;
+
+use Drupal\sapi\Plugin\StatisticsPluginInterface;
+
+/**
+ * @StatisticsPlugin(
+ *  id = "dummy",
+ *  label = "Dummy label"
+ * )
+ */
+class Dummmy implements StatisticsPluginInterface {
```

Comment:
it's actually optional to use the base class.  it is only required that it use the interface